start/src/main/liberty/config/server.xml

```diff
@@ -4,7 +4,7 @@
     <feature>jaxrs-2.0</feature>
     <feature>jsonp-1.0</feature>
     <feature>cdi-1.2</feature>
-    <feature>mpConfig-1.2</feature>
```

Comment:
Please don't remove the only config feature here, I removed the extra one last time already